src/dpp/dave/decryptor.h

```diff
@@ -0,0 +1,188 @@
+/************************************************************************************
+ *
+ * D++, A Lightweight C++ library for Discord
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright 2021 Craig Edwards and D++ contributors
+ * (https://github.com/brainboxdotcc/DPP/graphs/contributors)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * This folder is a modified fork of libdave, https://github.com/discord/libdave
+ * Copyright (c) 2024 Discord, Licensed under MIT
+ *
+ ************************************************************************************/
+#pragma once
+
+#include <array>
+#include <deque>
+#include <functional>
+#include <memory>
+#include <mutex>
+#include <vector>
+
+#include "codec_utils.h"
+#include "common.h"
+#include "cipher_interface.h"
+#include "cryptor_manager.h"
+#include "frame_processors.h"
+#include "version.h"
+#include "clock.h"
+
+namespace dpp::dave {
+
+class key_ratchet_interface;
+
+/**
+ * @brief Decryption stats
+ */
+struct decryption_stats {
```

Comment:
Do we really think any of these will go to uint64_t? I just don't really see the point (unless they physically can go that high).

Reply:
likely yes, discord set them this high and theyre packet counts, you can get millions of packets in a vc